player/command.c

```diff
@@ -884,6 +884,24 @@ static int mp_property_playback_time(void *ctx, struct m_property *prop,
     return property_time(action, arg, get_playback_time(mpctx));
 }
 
+static int mp_property_remaining_file_loops(void *ctx, struct m_property *prop,
+        int action, void *arg)
+{
+    MPContext *mpctx = ctx;
+    if (mpctx->remaining_file_loops == -1)
+        return m_property_double_ro(action, arg, INFINITY);
```

Comment:
We could return INT_MAX instead. I think returning bigger number for infinity is easier for downstream users to avoid special cases where simple compare would be enough.

Reply:
Or `10000` (+1) which is currently the max value for `loop-file`.

Reply:
I think -1 being infinity is an internal detail that script writers shouldn't have to care about, and having it positive is convenient to simplify comparisons, e.g. `mp.get_property_native('remaining-file-loops') > 0. And I think that having this be `math.huge` in Lua is convenient whereas `INT_MAX` is not available in Lua, or 10k is just an arbitrary number. Only with `math.huge` it is intuitive what to compare to to check for infinity in scripts. Also `--loop-file` is 10000 max but `--ab-loop-count` is `INT_MAX` max which is an inconsistency.

Reply:
I don't know what is the best approach here, but I agree with @na-na-hi that type changing is not great. It would be annoying to handle if used from C for example.

Reply:
Maybe always return a double?

Reply:
Updated to -1 but for what it's worth `--image-display-duration=inf` does map to `math.huge` in Lua and I use this and find it convenient.